locale/en_US/common.xml

```diff
@@ -453,6 +453,7 @@
 	<message key="notification.unsubscribeError">There was an error unsubscribing you.  Please ensure that you are using the correct URL as included in your notification email.</message>
 	<message key="notification.unsubscribeSuccess">You were successfully unsubscribed.</message>
 	<message key="notification.savedSubmissionMetadata">Submission metadata saved.</message>
+	<message key="search.author">By Author</message>
```

Comment:
Thanks for the note @mtub. I've moved over the locale string for all locales where it was found. However, some of the locales lacked a `common.xml` file in `pkp-lib`, so I left the `search.author` key alone in the OJS-specific `locale.xml` file. 

Not sure what should be done in this case, but here's the list of locales where it's still in the old place:

- ar_IQ
- fr_FR
- mk_MK
- no_NO
- pl_PL
- sk_SK
- te_ST